.github/workflows/build-plugin-archive.yml

```
@@ -147,6 +149,11 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: ${{ needs.checkout-dependencies.outputs.artifact }}
+      - name: Setup PHP with tools
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ inputs.PHP_VERSION_BUILD }}
+          tools: humbug/php-scoper:dev-main, rector
```

The previous conversation (https://github.com/inpsyde/reusable-workflows/pull/134#discussion_r1741806490) was hidden by merging #135; commenting this again so it doesn't get lost:
Is there a particular reason for using `dev-main`? Unreleased, buggy changes on that branch could lead to hard-to-detect bugs in the generated code.
```suggestion
          tools: humbug/php-scoper, rector

```
Also added a new line above the next step.

---

[There must have been a reason](https://github.com/inpsyde/reusable-workflows/pull/134/commits/946f4e425080eb9ab598334e8f0d2867f55d3960), but that was before the latest public release, so removing it should be fine - or at least worth trying